managed-databases/postgresql-and-mysql/how-to/enable-autobackup.mdx

```diff
@@ -3,44 +3,56 @@ meta:
   title: How to enable autobackup for Databases
   description: This page explains how to enable autobackup for Databases
 content:
-  h1: How to enable autobackup for Databases
+  h1: How to manage autobackups for Database Instances
   paragraph: This page explains how to enable autobackup for Databases
-tags: managed-dabatase autobackup database 
+tags: managed-dabatase autobackup database
 dates:
-  validation: 2023-01-19
+  validation: 2023-04-18
   posted: 2019-05-11
 categories:
   - managed-databases
   - postgresql-and-mysql
 ---
 
-Backups are logical data dumps of an individual database. They are useful when you need to restore a specific logical database. 
+With the autobackup feature the creation and management of your Database Instance backups is automated according to your backup schedule configuration.
```

Comment:
`With the autobackup feature, the creation and management of your Database Instance backups are automated according to your backup schedule configuration.`